manual/manual/cmds/comp.etex

```diff
@@ -188,6 +188,28 @@ to find by itself the ".cmo" file that implements a module with a
 given name: it relies instead on the user providing the list of ".cmo"
 files by hand.
 
+The {\em load path} is a list of directories and single files maintained by the
+compiler to look up, in order, for files needed to compile and/or link programs.
+
+By default the load path consists of the current directory and the standard
+library directory, but new entries can be added by using the "-I" and "-inc"
+flags.
+
+A directory can be added to the load path using "-I <dir>". Whenever the
+compiler needs to look up a file given by a relative path it will look up inside
+"<dir>". This includes the case of looking up top-level modules mentioned in the
+program text, but also relative paths passed on the command-line.
+
+A single file can also be added to the load path using "-inc <file>". This
+option is equivalent to "-I d" where "d" is a fresh directory containing only a
+single file with the same basename as "<file>". This option is meant mostly for
+programmatic use, and not by human developers.
+
+New entries added by using the "-I" and "-inc" flags are inserted, in order,
+after the current directory, but before the standard library directory. Search
+order in the load path is always left-to-right, so that earlier entries take
+precedence over later entries.
```

Comment:
This paragraph is partly redundant to the `By default, ...` paragraph in `-I`, which is itself in large part duplicated into the `By default, ...` paragraph in `-inc`. Can we remove the redundancy? (I envision problems if we update one of the three versions later.)